### ISSUE TITLE
Generic dataset "infrared absorption spectroscopy" does not recognize ATR-FTIR spectra

### ISSUE THREAD

**Issue by Thomas-Emmler-Hereon** [ComPlat/chemotion_ELN#1620]
- I am an ELN admin

Description
When importing ATR-FTIR (.dx file from Bruker FTIR) spectra into the Analyses tab, the spectra are displayed in transmission. 

There may be no evaluation of the .dx file header when displaying the spectrum in the Spectra Editor (##SAMPLING PROCEDURE=ATR eco Ge; ##YUNITS=ABSORBANCE).

The content section of the spectrum in Chemotion correctly states: "IR (ATR, ṽ) =..."

When importing the .dx file, I would expect the spectrum to be displayed in absorbance. An alternative would possibly be a corresponding switching option in the Spectra Editor.

...Thomas

**Comment by nicolejung** [comment]
Very good point - we need to adapt the process. 
Option one: we could use IR (by ontology) in combination with a selection of absorbance/transmission for y-axis
Option two: select directly IR transmission/IR absorbance as subterm for IR with fixed y description

**Comment by Thomas-Emmler-Hereon** [comment]
The more I think about it, the more I think that there should be a separate ontology for ATR FTIR. 

Whereas in transmission measurements the path length is always defined by the sample thickness, the penetration depth in ATR measurements is wavelength-dependent (and thus the intensities in the spectrum are also, but not only, different from those in transmission measurements) - any other representation of the spectrum then raises the question of whether an ATR correction has already been applied or not... Without knowledge of the refractive index and the geometry of the ATR unit, ATR correction is only possible to a limited extent using an intensity ramp, for example. That would require quite far-reaching changes in Chemotion or spectra, I guess.

I therefore think that a separate ontology (or a corresponding special sub-term) for the ATR-FTIR measurements would be advantageous (including a selection button to indicate whether the ATR correction has already been applied and possibly a text field indicating which software was used for this).

**Comment by nicolejung** [comment]
yes, sorry, I stopped with the two methods given above but there are more options one could select: e.g. for ATR: attenuated total reflectance Fourier transform infrared spectroscopy (ATR-FTIR). 
You could have a look here: https://service.tib.eu/ts4tib/ontologies/chmo and search for the possible options. -> and we could have a quick call about the best choice (if possible)
